db/filename_test.cc

```diff
@@ -171,6 +171,24 @@ TEST_F(FileNameTest, Construction) {
   ASSERT_EQ(kMetaDatabase, type);
 }
 
+TEST_F(FileNameTest, NormalizePath) {
+
+  std::string separator(1, kFilePathSeparator);
+  std::stringstream given, toNormalize;
+
+  given << separator << separator << "SERVER" << separator << "fileName";
+  toNormalize << separator << "pathel" << separator << "pathel_1";
+
+  std::string expected = given.str();
+
+  ASSERT_EQ(expected, NormalizePath(given.str()));
+
+  expected = toNormalize.str();
+
+  ASSERT_EQ(expected, NormalizePath(toNormalize.str()));
+
```

Comment:
You have //a/b and /a/b as test cases, can you add a few more:
a/b -- no leading slash
/a (two characters)
a/ (two characters)
/ (only a slash)
// (only a UNC).

Where "/" is kFilePathSeparator.

Reply:
ok

Reply:
ok. added